Makefile

```diff
@@ -52,6 +52,8 @@ lint :
 	done;
 
 test : build
+	@echo "== build fake nginx for tests"
+	@go build -o nginx/fake/fake_graceful_nginx nginx/fake/fake_graceful_nginx.go
```

Comment:
I think this should go in its own target, so that you can build the fake nginx binary and then run the test in your IDE.
the test target would then depend on that new target